extensions/data-plane/data-plane-google-storage/README.md

```diff
@@ -5,26 +5,35 @@
 This module contains a Data Plane extension to copy data to and from Google cloud storage.
 
 ### Authentication
+Google storage data plane supports three different approaches for authentication:
+* Default authentication:
+    * Authenticates against the Google Cloud API using the [Application Default Credentials](https://cloud.google.com/docs/authentication#adc).
+    * These will automatically be provided if the connector is deployed with the correct service account attached.
+* Service Account key file
```

Comment:
Suggest to add a link about managing key files : https://cloud.google.com/iam/docs/best-practices-for-managing-service-account-keys